packages/example/src/components/remote-control/RemoteControlManager.ios.ts

```diff
@@ -19,12 +19,18 @@ class RemoteControlManager implements RemoteControlManagerInterface {
       up: SupportedKeys.Up,
       down: SupportedKeys.Down,
       select: SupportedKeys.Enter,
+      longSelect: SupportedKeys.LongEnter,
     }[evt.eventType];
 
     if (!mappedKey) {
       return;
     }
 
+    // We only want to handle keydown fon long select
```

Comment:
"because it delays input and we don't want to add lag on all inputs"

(is that correct?)

Reply:
It is because we don't want to handle twice the long select (when keydown and when keyup)

Reply:
I will refine the comment